frontend/src/pages/Home.jsx

```
@@ -92,16 +116,16 @@ const Home = () => {
-          {[
-            { id: "business-suite", name: "Business Suite", price: "$120/night", icon: <Building size={40} /> },
-            { id: "luxury-room", name: "Luxury Room", price: "$250/night", icon: <Building size={40} /> },
-            { id: "standard-room", name: "Standard Room", price: "$80/night", icon: <Building size={40} /> }
-          ].map((room) => (
-            <div key={room.id} className="bg-white shadow-md rounded-lg p-6 text-center hover:shadow-lg transition">
-              <div className="flex justify-center mb-4 text-blue-600">{room.icon}</div>
-              <h3 className="text-xl font-semibold mb-2">{room.name}</h3>
-              <p className="text-gray-600">{room.price}</p>
-            </div>
+          {properties.map((property) => (
+            <Link to={`/property/${property._id}`}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)